crates/runtime/src/embeddings/array_distance.rs

```diff
@@ -51,9 +48,10 @@ impl Default for ArrayDistance {
 /// For two [`DataType::FixedSizeList`], the inputs must have the same length, and have compatible
 /// inner types. Compatible inner types are
 ///   - Both inputs are [`DataType::Float16`], [`DataType::Float32`], or [`DataType::Float64`].
```

Comment:
Float16 should be removed from comment if we don't support it in these functions anymore